radicle-cob/src/object/collaboration/update.rs

```diff
@@ -93,5 +93,5 @@ where
         change.timestamp,
     );
 
-    Ok(object)
+    Ok((object, change.id))
```

Comment:
This bubbles up and eventually gets returned from `Transaction::commit` as `EntryId` (previously `OpId`). Unless I'm mistaken, the `change.id` is the commit oid of the change, ie. the op bundle.

Reply:
Right, that makes sense! Just wanted to double check :) I wonder if we can use `EntryId` as the return type then? And/or have a struct for the return type. I have an aversion to tuples because they tend to give less of a hint as to what this thing is.

Reply:
I wanted to, but we don't have Entry there, just Change... I've been meaning to somehow reconcile these two types as they are so similar but seems like more work.

I'm not sure what the struct would be called, I guess `Updated {}`?

Reply:
Done.